MIGRATION.MD

```diff
@@ -20,6 +20,9 @@ In order to make this breaking change as backward compatible as possible, the de
 This is as close as possible to the [`smart` merge strategy](https://github.com/survivejs/webpack-merge/tree/v4.2.2#smart-merging) that was used before, so ideally if you didn't use `mergeStrategies` then your configuration should work just as it worked before.  
 Otherwise you'll have to adjust the `mergeRules` in accordance with your needs.
 
+## Jest builder
+1. `jest-preset-angular` has been updated to version 9, which uses Angular compiler instead of `ts-jest` in order to transform the TS files. Make sure you understand the implications and perform all the necessary changes to your code base as described in `jest-preset-angular` [CHANGELOG](https://github.com/thymikee/jest-preset-angular/blob/master/CHANGELOG.md).
```

Comment:
Hi, `jest-preset-angular` have reverted that change and will still use `ts-jest` for `isolatedModules: false`. With `isolatedModules: true`, codes will be transformed via `jest-preset-angular` instead of `ts-jest`.

Reply:
@ahnpnl thanks for noticing. Anyways this update will have to wait until they release stable version `9`.

Reply:
I'm still waiting for Jest 27. Once Jest 27 is out, version 9 will follow too.

Reply:
**9.0.0** was out.

Reply:
can we try to update to the latest version now and also update the `MIGRATION.MD`? [jest-preset-angular@9.0.1](https://github.com/thymikee/jest-preset-angular/releases/tag/v9.0.1)

Reply:
@just-jeb  any update on this? still have to use plain jest to run angular tests

Reply:
I've pushed changes according to the breaking changes from the [CHANGELOG](https://github.com/thymikee/jest-preset-angular/blob/master/CHANGELOG.md), if it passes I'll update the migration guide and merge.  
However, if there are issue I won't be able to investigate properly since I'm not away from computer until June 20th.  

You're welcome to help though if there are issues with the build/tests - while I can't investigate I can **release** a new version from my phone 😄